infra/fastly/terraform/dl-sandbox.k8s.dev/vcl/binaries.vcl

```diff
@@ -44,8 +44,15 @@ sub vcl_fetch {
     set beresp.stale_while_revalidate = 60s; // 1 minute
   }
 
-  //Ensure version markers are not cached
-  if (req.url.ext == "txt") {
+  # Ensure version markers are not cached at the edge
+  if (req.url.path ~ "^/release/(latest|stable)(-\d+(\.\d+))?\.txt\z") {
+    set beresp.cacheable = false;
+    set beresp.ttl = 0s;
+    return (pass);
+  }
+
+  # Ensure HTML and JSON files are not cached at the edge
+  if (req.url.ext ~ "(html|json)\z") {
```

Comment:
FYI @kubernetes/release-engineering, something to keep in mind, if you see artifacts that should or shouldn't be cached (e.g. if releases start including html/json files) we need to refine these rules and deploy them (not automated yet)